python/ray/tune/commands.py

```diff
@@ -233,6 +248,16 @@ def list_experiments(project_path,
 
     print_format_output(info_df)
 
+    if output:
+        output_path = os.path.join(base, output)
+        if output.split('.')[-1].lower() in ["p", "pkl", "pickle"]:
```

Comment:
You could actually extract the file extension using the `os.path.splitext`:
```python
file_extension = os.path.splitext(output)[1]
if file_extension in ("p", "pkl", "pickle"):
    ...
elif file_extension == "csv":
    ...
else:
    ...
```

Reply:
that's helpful, thanks!